packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -337,6 +353,81 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     this.setState({ menuProps: currentMenuProps ? null : menuProps });
   }
 
+  @autobind
```

Comment:
I'm honestly still not really familiar with those decorators.
Don't I need to bind `this` since I'm using it later on for `this.props...`?